src/week05/project/index.rst

```diff
@@ -77,6 +77,19 @@ For your secondary objectives no guidance will be given to you. You will have to
 - Database is secure from SQL injection.
 - Add coordinate information to CSV files that are missing latitude and longitude.
 
+Bonus Missions
+==============
+
+If you finish all objectives above, here are some additional features to consider. These are roughly listed in order from easiest to hardest. Feel free to pick what seems interesting to you, rather than starting from the top of the list. These are all independent of one another. 
+
+- Remove dependence on jQuery
```

Comment:
should we strip jQuery from the curriculum in general? in the prep weeks we skipped jQuery and i showed them native js DOM methods and fetch instead.